src/include/stat.h

```diff
@@ -1138,19 +1159,26 @@ struct __wt_dsrc_stats {
     int64_t checkpoint_cleanup_pages_walk_skipped;
     int64_t checkpoint_cleanup_pages_visited;
     int64_t checkpoint_obsolete_applied;
-    int64_t compress_precomp_intl_max_page_size;
-    int64_t compress_precomp_leaf_max_page_size;
-    int64_t compress_read;
-    int64_t compress_write;
-    int64_t compress_hist_ratio_max;
-    int64_t compress_hist_ratio_16;
-    int64_t compress_hist_ratio_2;
-    int64_t compress_hist_ratio_32;
-    int64_t compress_hist_ratio_4;
-    int64_t compress_hist_ratio_64;
-    int64_t compress_hist_ratio_8;
     int64_t compress_write_fail;
     int64_t compress_write_too_small;
+    int64_t compress_read;
+    int64_t compress_read_ratio_hist_max;
+    int64_t compress_read_ratio_hist_2;
+    int64_t compress_read_ratio_hist_4;
+    int64_t compress_read_ratio_hist_8;
+    int64_t compress_read_ratio_hist_16;
+    int64_t compress_read_ratio_hist_32;
+    int64_t compress_read_ratio_hist_64;
+    int64_t compress_write;
+    int64_t compress_write_ratio_hist_max;
+    int64_t compress_write_ratio_hist_2;
+    int64_t compress_write_ratio_hist_4;
+    int64_t compress_write_ratio_hist_8;
+    int64_t compress_write_ratio_hist_16;
+    int64_t compress_write_ratio_hist_32;
+    int64_t compress_write_ratio_hist_64;
+    int64_t compress_precomp_intl_max_page_size;
```

Comment:
The A->Z sorting seems wrong here

Reply:
Yep that's because this is auto-generated by `dist/stat.py` and the script sorts on [description text instead of variable name](https://github.com/wiredtiger/wiredtiger/blob/b311ca93ee650984e9104a250efb3e7bf431ac6c/dist/stat.py#L38). It was one of the motivations for rewording descriptions to give us nicer ordering of variables.